### PR TITLE
Adds support for the Bitski browser extension wallet

### PR DESCRIPTION
**Pull request by pzuraq**
## Description

Add support for Bitski's new browser extension wallet

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address: 0xb9b7d3d6317d987548919641c8bf1f539da275fc

### CODE REVIEW

**Comment by pzuraq** [comment]
I don't believe the test failure was related to this change but I could be wrong, let me know if I am

File: .changeset/purple-ligers-jog.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/connectors': minor
+---
+
+Add support for Bitski injected wallet
```
Comment (tmm) [inline .changeset/purple-ligers-jog.md:5]:
```suggestion
Added name mapping for Bitski
```

File: .changeset/purple-ligers-jog.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/connectors': minor
```
Comment (tmm) [inline .changeset/purple-ligers-jog.md:2]:
```suggestion
'@wagmi/connectors': patch
```

**Review by tmm — CHANGES_REQUESTED** [review]
Looking good. Please update the test `getInjectedName.test.ts` for `isBitski`.

**Comment by pzuraq** [comment]
@tmm updated!

**Pull request merged**